### PR TITLE
Bring master changes to network-ng

### PR DESCRIPTION
**Pull request by imobachgs**
Bring master changes to the networ-ng branch.

### CODE REVIEW

File: test/new_device_startmode_test.rb [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+require "yast"
+
+Yast.import "ProductFeatures"
+Yast.import "LanItems"
+
+describe "LanItemsClass#new_device_startmode" do
```
Comment (jreidinger) [inline test/new_device_startmode_test.rb]: this is wrong as this method is no longer in network-ng, or am I wrong?

File: test/new_device_startmode_test.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env rspec
```
Comment (teclator) [inline test/new_device_startmode_test.rb]: This test was completely removed in network-ng

File: test/new_device_startmode_test.rb (hunk 1 above)
Comment (imobachgs) [reply test/new_device_startmode_test.rb]: Yes, you are right. Dropped.

File: test/new_device_startmode_test.rb (hunk 2 above)
Comment (imobachgs) [reply test/new_device_startmode_test.rb]: Yes, you are right. Dropped.

**Comment by teclator** [comment]
@imobachgs @jreidinger there is a failing test because Commit expect now the builder. I can fix it later if you want as part of the CLI adaptions I'm doing for network-ng

**Comment by teclator** [comment]
LGTM

**Pull request merged**